src/System.Net.HttpListener/src/System/Net/HttpResponseStream.cs

```diff
@@ -40,6 +40,19 @@ public override IAsyncResult BeginRead(byte[] buffer, int offset, int size, Asyn
         }
 
         public override int EndRead(IAsyncResult asyncResult) => throw new InvalidOperationException(SR.net_writeonlystream);
+        
+        public new Task<int> ReadAsync(Byte[] buffer, int offset, int count)
```

Comment:
You don't need to provide this overload of `ReadAsync` here. `Stream` [already provides](https://github.com/dotnet/coreclr/blob/6a384f32c7db1a65fae5414da55cd993d42df756/src/mscorlib/src/System/IO/Stream.cs#L405-L408) a non-virtual implementation that just calls the virtual overload that takes a `CancellationToken`, passing `CancellationToken.None`.